selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java

```diff
@@ -101,9 +101,8 @@ public void goToTaskPage() {
    */
   public void goToMySpaces() {
     info("-- Go to my spaces --");
-    ELEMENT_MY_SPACE_LINK_PLF.waitUntil(Condition.visible, Configuration.timeout);
     sleep(2000);
-    ELEMENT_MY_SPACE_LINK_PLF.click();
+    ELEMENT_MY_SPACE_LINK_PLF.waitUntil(Condition.visible,Configuration.timeout).click();
```

Comment:
hanks to this change we can remove the sleep on the line above and below, no ?

Reply:
Done.